pygmt/datasets/earth_relief.py

```diff
@@ -103,12 +118,18 @@ def load_earth_relief(resolution="01d", region=None, registration=None):
             f"resolution '{resolution}' is not supported."
         )
 
+    # Choose earth relief data prefix
+    earth_relief_prefix = "earth_relief_"
+    if resolution in land_only_srtm_resolutions:
+        if use_srtm:
+            earth_relief_prefix = "srtm_relief_"
```

Comment:
The same but shorter.
```suggestion
    if resolution in land_only_srtm_resolutions and use_srtm:
        earth_relief_prefix = "srtm_relief_"
```